Index: AgendaTelefonica/src/Menu.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>public class Menu {\r\n\r\n\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/AgendaTelefonica/src/Menu.java b/AgendaTelefonica/src/Menu.java
--- a/AgendaTelefonica/src/Menu.java	(revision a0aa95291c2b765b644e5311590f08df46ef8830)
+++ b/AgendaTelefonica/src/Menu.java	(date 1738697721086)
@@ -1,4 +1,5 @@
 public class Menu {
+    
 
 
 
Index: AgendaTelefonica/src/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>public class Main {\r\n    public static void main(String[] args) {\r\n        System.out.println(\"Hello, World!\");\r\n\r\n\r\n\r\n\r\n\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/AgendaTelefonica/src/Main.java b/AgendaTelefonica/src/Main.java
--- a/AgendaTelefonica/src/Main.java	(revision a0aa95291c2b765b644e5311590f08df46ef8830)
+++ b/AgendaTelefonica/src/Main.java	(date 1738697721098)
@@ -2,9 +2,5 @@
     public static void main(String[] args) {
         System.out.println("Hello, World!");
 
-
-
-
-
     }
 }
\ No newline at end of file
